crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -639,9 +636,9 @@ struct LogsControllerInner {
 }
 
 impl LogsControllerInner {
-    fn new(configuration: LogsConfiguration, retry_policy: RetryPolicy) -> Self {
+    fn new(retry_policy: RetryPolicy) -> Self {
         Self {
-            current_logs: Arc::new(Logs::with_logs_configuration(configuration)),
+            current_logs: Arc::new(Logs::from_configuration(&Configuration::pinned())),
```

Comment:
After merging #2452, this should be `Metadata::with_current(|m| m.logs_snapshot())` unless I'm missing something.